target/linux/ath79/image/generic.mk

```diff
@@ -1479,6 +1479,12 @@ define Device/plasmacloud_pa300
 endef
 TARGET_DEVICES += plasmacloud_pa300
 
+define Device/plasmacloud_pa300e
+  $(Device/plasmacloud_pa300)
```

Comment:
Please do not inherit one device from another. Create a separate definition for the common properties instead and then inherit both devices from that. There should be a lot of examples in generic.mk